### PR TITLE
Use two byte reference number when submitting long messages

### LINKED ISSUE #45: Support 2 byte reference number in UDH header
Hey there, 

We're going to be running some services with this library that are going to be using a fairly large number of binds. With the current 1 byte reference number the chance of a collision would be relatively high and we'd like to use the alternative header with the 2 byte number. For now we've patched the library to add it in but was wanting to ask if you would accept a PR for it?

Cheers,
Lucian

### PR DESCRIPTION
**Pull request by lucianjon**
to fix #45

### CODE REVIEW

File: smpp/transmitter.go [hunk 1]
```diff
@@ -327,21 +327,25 @@ func (t *Transmitter) Submit(sm *ShortMessage) (*ShortMessage, error) {
...
+	maxLen := 133 // 140-7 (UDH with 2 byte reference number)
+	if sm.Text.Type() == pdutext.UCS2Type {
+		maxLen = 132 // to avoid a character being split between payloads
+	}
 	rawMsg := sm.Text.Encode()
 	countParts := int((len(rawMsg)-1)/maxLen) + 1
 
 	t.rMutex.Lock()
-	ri := uint8(t.r.Intn(128))
+	rn := uint16(t.r.Intn(65535))
```
Comment (therealplato) [inline smpp/transmitter.go]: `t.r.Intn(0xFFFF)` may be a little more clear

Comment (fiorix) [inline smpp/transmitter.go]: Totally

**Comment by fiorix** [comment]
Only that lil nit and should be good to go.

**Comment by lucianjon** [comment]
Sweet, have made the change

**Comment by fiorix** [comment]
Cool 👍

**Pull request merged**